src/integrationtest/python/GeoAttribute_tests.py

```diff
@@ -0,0 +1,48 @@
+# Name: Debasish Kanhar
+
+import unittest
+from subprocess import Popen, PIPE
+from JanuaGraphContainer import JanusGraphContainer
+from janusgraph_python.driver.ClientBuilder import JanusGraphClient
+from gremlin_python.structure.graph import Graph
+from janusgraph_python.core.attribute.Geo import Geo
+from janusgraph_python.core.datatypes.GeoShape import GeoShape
+
+
+class TestGeoAttributes(unittest.TestCase):
+    def setUp(self):
+        self.container = JanusGraphContainer()
+
+        docker_ip = Popen(["docker-machine", "ip"], stdout=PIPE).communicate()[0]
+        docker_ip = docker_ip.strip().decode("utf-8")
+
+        self.client = JanusGraphClient()
+        self.client = self.client.connect(host=str(docker_ip), port="8182",
+                                          traversal_source="gods_traversal").get_connection()
+
+        self.container.start()
+        self.g = Graph().traversal().withRemote(self.client)
+
+    def tearDown(self) -> None:
+        self.client.close()
+        self.container.stop()
+
+    def test_geo_contains(self):
+
+        # First we will need to add a property so that GeoContains can be queried.
+        place = GeoShape.Circle(50, 100, 100)
+        self.g.V().has("name", "tartarus").property("place", place).next()
+
+        truth = {GeoShape.Circle(50, 100, 10): 1, GeoShape.Circle(50, 100, 500): 0}
```

Comment:
Personally Id prefer a name like `mock_data` to `truth` just so when you iterate below it would read more like a sentence. Updating this is totally up to you and is just a suggestion based off my opinion. 

example: `for k, v in mock_data.items()` When I look at this I literally read in my head, "for keys and values in mock data items"